ml_metadata/tools/mlmd_bench/thread_runner_test.cc

```diff
@@ -54,6 +54,12 @@ void TestThreadRunner(const int num_thread) {
   // Checks that the workload indeed be executed by the thread_runner.
   EXPECT_EQ(get_response.artifact_types_size(),
             mlmd_bench_config.workload_configs()[0].num_operations());
+
+  // Checks for valid performance report.
```

Comment:
nit, use a summary = benchmark.mlmd_bench_report().summaries()[0], then use the variable in the two EXPECT below

often to use [0], we need to add a 

ASSERT_THAT(::testing::SizeIs ... ) or ASSERT_EQ(...size(), ?) to ensure using [0] is safe. 

ASSERT fails the test at this line, EXPECT will continue run the rest of the lines.